data/coalition jobs.txt

```diff
@@ -63,7 +63,7 @@ mission "Coalition: Ringworld Workers 3"
 		"coalition jobs" ++
 		payment
 		payment 20000
-		dialog `You drop off the volunteers on the <planet>. A few minutes later you receive a transfer of <payment>.`
+		dialog `You drop off the volunteers on the <planet>. A few minutes later, you receive a transfer of <payment>.`
```

Comment:
@jafdy @strollcata It is a style choice, and since MZ has his own style I would try to preserve it.

https://www.grammarly.com/blog/commas-after-introductory-clauses/

>As with many other rules in English grammar, the comma-after-introductory-clauses rule comes with an exception. When the introductory clause is short, the comma may be omitted as a matter of style.
>See the example below:
>[Grabbing her umbrella Kate raced out of the house.]
>Since the introductory clause consists of only three words, the comma separating the introductory clause from the main clause may or may not be used.